### PR TITLE
Adx 973 override frictionless framework badges with our own

### PR DESCRIPTION
**Pull request by A-Souhei**
## Description

Updates validation badges to ADR badges
Moved the badges code unto UNAIDS

## Checklist

Put an `x` in the boxes that apply to this pull request (you can also fill these out after opening the pull request).
You may not need to check all boxes.

- [x] The Jira ticket for this issue has been updated to "Ready to Review" or equivalent.
- [x] I have developed these changes in discussion with the appropriate project manager.
- [ ] My code follows the general Fjelltopp documentation (see Confluence).
- [ ] I have made corresponding changes to the Fjelltopp documentation (see Confluence).
- [ ] I have rebased this branch with master.
- [ ] New dependency changes have been committed.
- [ ] I have added automated tests that prove my fix is effective or that my feature works.
- [x] New and existing tests pass locally with my changes.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own code.
- [x] I have assigned at least one reviewer.

### CODE REVIEW

File: ckanext/unaids/theme/public/images/badges/data-error-flat.svg
Comment (ChasNelson1990) [inline ckanext/unaids/theme/public/images/badges/data-error-flat.svg:1]: So I am not sure we need these old badges if we have the new badges working?

File: ckanext/unaids/assets/js/validation-badge.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                success: function(data){module._success(data);},
+                error: function(XMLHttpRequest, textStatus, errorThrown){module._error(XMLHttpRequest, textStatus, errorThrown);},
+                dataType: "json",
+                complete: setTimeout(function(){module._complete();}, 5000),
+                timeout: 2000
+            });
+        },
+        _success: function(data) {
+            this.options.status = data.result.status;
+            if(this.options.status != 'running' && this.options.status != 'created'){
```
Comment (ChasNelson1990) [inline ckanext/unaids/assets/js/validation-badge.js]: This doesn't seem to work at all for me - if I have a validation that is running (e.g. a large geojson) then I get no badge (which is the upstream behaviour)

File: ckanext/unaids/assets/webassets.yml [hunk 2]
```diff
@@ -64,3 +64,14 @@ MapSelectorCSS:
     filters: cssmin
     output: unaids/%(version)s_MapSelector.css
 
+main_css:
```
Comment (ChasNelson1990) [inline ckanext/unaids/assets/webassets.yml]: I think we need to rename things here? Maybe ValidationBadgeCSS and ValidationBadgeJS?

File: ckanext/unaids/assets/webassets.yml [hunk 3]
```diff
@@ -64,3 +64,14 @@ MapSelectorCSS:
...
+    output: validation/%(version)s_main.css
```
Comment (ChasNelson1990) [inline ckanext/unaids/assets/webassets.yml]: I think this should be `unaids/%(version)s_ValidationBadge.css`

Comment (ChasNelson1990) [reply ckanext/unaids/assets/webassets.yml]: For consistency with other CSS assets

File: ckanext/unaids/assets/webassets.yml [hunk 4]
```diff
@@ -64,3 +64,14 @@ MapSelectorCSS:
...
+    contents:
+        - css/validation.css
+
+badge_js:
+    contents:
+      - js/validation-badge.js
```
Comment (ChasNelson1990) [inline ckanext/unaids/assets/webassets.yml]: I think this should be `unaids/%(version)s_ValidationBadge.js` for consistency with other JS assets

**Review by ChasNelson1990 — CHANGES_REQUESTED** [review]
The code changes look like a good starting point @A-Souhei - unfortunately it doesn't seem to work on my set up. I think a few things need changing slash adding.

E.g. there is no /templates/package/resource_read.html?

File: ckanext/unaids/theme/public/images/badges/data-error-flat.svg
Comment (A-Souhei) [reply ckanext/unaids/theme/public/images/badges/data-error-flat.svg:1]: image
is this the right badge?

File: ckanext/unaids/assets/js/validation-badge.js (hunk 1 above)
Comment (A-Souhei) [reply ckanext/unaids/assets/js/validation-badge.js]: Could you share with me a working geojson file so I can test it please?

**Review by ChasNelson1990 — CHANGES_REQUESTED** [review]
@A-Souhei - the Lint action isn't passing.

**Pull request merged**